android/javatests/org/chromium/chrome/browser/BytecodeTest.java

```diff
@@ -771,12 +774,26 @@ public void testFieldsExist() throws Exception {
                 "org/chromium/chrome/browser/ntp/NewTabPageLayout", "mMvTilesContainerLayout"));
         Assert.assertTrue(
                 fieldExists("org/chromium/chrome/browser/dom_distiller/ReaderModeManager", "mTab"));
+
         Assert.assertTrue(fieldExists(
                 "org/chromium/chrome/browser/omnibox/suggestions/DropdownItemViewInfoListBuilder",
                 "mDropdownHeight"));
         Assert.assertTrue(fieldExists(
                 "org/chromium/chrome/browser/omnibox/suggestions/DropdownItemViewInfoListBuilder",
                 "mPriorityOrderedSuggestionProcessors"));
+
+        Assert.assertFalse(fieldExists(
```

Comment:
It's more important to check whether these fields still exist in the upstream's `SearchEngineAdapter` as we don't control their code.

Reply:
Sorry, this is an accidental leftover

Reply:
thanks, fixed that at
https://github.com/brave/brave-core/pull/14402/commits/22d3cd2546b3d11d11a1785792e4d6482e83dee8#diff-b2d64845479ca6df40c36fb252689f1ebf820afd2e1b2f80bd684e489ab64f38R788